mailjet.cron.php

```diff
@@ -295,6 +295,10 @@ function findCode($code)
 					$content = str_replace('{'.$key.'}', $value, $content);
 				}
 
+                $content = str_replace('{shop_name}', $shop_name, $content);
+				$content = str_replace('{shop_url}', $shop_url, $content);
+				$content = str_replace('{shop_logo}', $shop_logo, $content);
+
```

Comment:
If you wish you can add me to the skype - ferhan_ismailov to make 
communication faster.

regards!

On 11/16/2015 14:21 PM, Ferhan Ismailov wrote:
> Nop, this comment is fixed and the fix is added to the PR
>
> I mention hat is OK for us to not have the fix in upgrade script for 
> the https://github.com/PrestaShop/mailjet/pull/24/files#r44911737 comment
>
> If there is something unclear don`t hesitate to contact me again. Have 
> a nice day!
>
> On 11/16/2015 14:18 PM, Thomas N wrote:
>>
>> In mailjet.cron.php 
>> <https://github.com/PrestaShop/mailjet/pull/24#discussion_r44916892>:
>>
>> > @@ -295,6 +295,10 @@ function findCode($code)
>> >  					$content = str_replace('{'.$key.'}', $value, $content);
>> >  				}
>> >
>> > +                $content = str_replace('{shop_name}', $shop_name, $content);
>> > +				$content = str_replace('{shop_url}', $shop_url, $content);
>> > +				$content = str_replace('{shop_logo}', $shop_logo, $content);
>> > +
>>
>> Hi,
>>
>> Thanks for this new commit.
>> However I do not understand what you mean for the upgrade. Do you 
>> mean you won't add it or you are going to commit it later ?
>>
>> —
>> Reply to this email directly or view it on GitHub 
>> <https://github.com/PrestaShop/mailjet/pull/24/files#r44916892>.
>>
>